config/config.go

```diff
@@ -46,42 +46,55 @@ const Mainnet protocol.NetworkID = "mainnet"
 const GenesisJSONFile = "genesis.json"
 
 // Local holds the per-node-instance configuration settings for the protocol.
+// !!! WARNING !!!
+//
+// These versioned struct tags need to be maintained CAREFULLY and treated
+// like UNIVERSAL CONSTANTS - they should not be modified once committed.
+//
+// New fields may be added to the Local struct, along with a version tag
+// denoting a new version. When doing so, also update the
+// installer/config.json.example and test/testdata/configs/config-v{n}.json
```

Comment:
I wonder if these can be autogenerated based on the values defined here...

Reply:
yes, I plan to do so. I will update this PR with these changes.